lib/src/blocs/auth_bloc/auth_state.dart

```diff
@@ -0,0 +1,21 @@
+part of 'auth_bloc.dart';
```

Comment:
We shouldn't use "part of" instead of import regularly

Reply:
This file has been removed